src/storage/FileserverFileLoader.cpp

```diff
@@ -1,29 +1,47 @@
 #include "FileserverFileLoader.h"
 
+#include <cpprest/astreambuf.h>
```

Comment:
This file contains all the changes to the HTTP client used in the fileserver client